third_party/terraform/website/docs/r/google_service_account_iam.html.markdown

```diff
@@ -109,6 +154,21 @@ The following arguments are supported:
 * `policy_data` - (Required only by `google_service_account_iam_policy`) The policy data generated by
   a `google_iam_policy` data source.
 
+* `condition` - (Optional, [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) An [IAM Condition](https://cloud.google.com/iam/docs/conditions-overview) for a given binding.
+  Structure is documented below.
+
+The `condition` block supports:
+
+* `expression` - (Required) Textual representation of an expression in Common Expression Language syntax.
+
+* `title` - (Required) A title for the expression, i.e. a short string describing its purpose.
+
+* `description` - (Optional) An optional description of the expression. This is a longer text which describes the expression, e.g. when hovered over it in a UI.
+
+~> **Warning:** Terraform considers the `role` and condition contents (`title`+`description`+`expression`) as the
```

Comment:
Do we want `description` as part of the identifier for the binding?

Reply:
Yes and no. It feels silly to do so, but it also makes the code much easier to deal with, and it aligns with what the IAM team has told us (since there can be two distinct conditions with the exact same title and expression but different descriptions). Since a ForceNew on an IAM  resource isn't super destructive, I'm pretty ok with leaving it this way and seeing if it causes any issues.